Adding swoop entrance and exit animations.

emmanuelulloa opened the pull request:
I added a set of swooping entrance and exit animations.
They are inspired on the "smear" effect by Chuck Jones.
Review them in this codepen: [https://codepen.io/emmanuelulloa/pen/PogVYPa](https://codepen.io/emmanuelulloa/pen/PogVYPa)

emmanuelulloa committed:
Adding swoop entrance and exit animations.

emmanuelulloa committed:
Changed 0% to "from" and 100% to "to"

emmanuelulloa commented:
@daneden , @eltonmesquita o @warengonzaga any comment on the animations?

warengonzaga commented:
Hello @emmanuelulloa, I like the animation, but is there anything we can do to make it more fluid? The final decision would be from @eltonmesquita.

daneden commented:
@emmanuelulloa these are great! I’d suggest adding an opacity transition in the final ~50% of each keyframe animation. The translation is quite small and it's unlikely that transitioned elements would appear “off-screen” by the end of the animation (unlike some of the current exit/entrances, which translate elements by thousands of pixels to ensure they begin/end off screen).

emmanuelulloa committed:
Changed opacity change position.

As suggested by Dan Eden, I changed the opacity value change to last longer so is more visually perceived when the element enters and leave.

emmanuelulloa commented:
@daneden I added the opacity change and you are correct; they look way more natural.  Please review at the same codepen: https://codepen.io/emmanuelulloa/pen/PogVYPa

daneden commented:
@emmanuelulloa nice, this does look better, although I'd suggest the opacity transition is a bit abrupt. It looks like the transition starts at the 65% keyframe right now; I’d play around with moving it to the 51% or 57% keyframe.

emmanuelulloa committed:
Moved the opacity change to a more middle position.

emmanuelulloa commented:
@daneden I move it to 57% (because at 51% it looked more like a regular fade).  Please review and let me know what you think.

emmanuelulloa commented:
@daneden le me know what you think of the latest fix.  Regards.

warengonzaga commented:
Hey @eltonmesquita can you take a look into this? Thanks!

emmanuelulloa commented:
@eltonmesquita any comments on regards to this one?

emmanuelulloa commented:
@daneden any chance you see into this before the end of 2024?

daneden reviewed (approved):
This looks great, thanks for your adjustments! Approving to unblock. @eltonmesquita @warengonzaga you're both more familiar these days with the release process so I’ll leave it to you to approve/merge.

warengonzaga commented:
> This looks great, thanks for your adjustments! Approving to unblock. @eltonmesquita @warengonzaga you're both more familiar these days with the release process so I’ll leave it to you to approve/merge.

Thanks! I'm not sure if I have merge capability but I will do my best to help approve the PRs.

emmanuelulloa commented:
@warengonzaga let me know if you need something from me when doing the merge.

emmanuelulloa commented:
@eltonmesquita let me know if you need something from me for your review process.

emmanuelulloa commented:
@daneden @warengonzaga @eltonmesquita will the merge happen at any time in the near future?

warengonzaga commented:
@eltonmesquita, are we going to merge this? Sorry @emmanuelulloa I don't have merge capabilities.
